pkg/storage/secret/encryption/data/data_key_delete.sql

```diff
@@ -0,0 +1,5 @@
+DELETE FROM {{ .Ident "secret_data_key" }}
+WHERE 1 = 1 AND
```

Comment:
think we can remove the "1 = 1" from most of the queries, since is not actually needed, and as was reading it might actually make queries slower because it will not use some indexes. we can do this in another PR to also tackle the ones that are already merged